packages/client/src/index.ts

```diff
@@ -707,6 +711,110 @@ export async function connectScryptedClient(options: ScryptedClientOptions): Pro
             .map(id => systemManager.getDeviceById(id))
             .find(device => device.pluginId === '@scrypted/core' && device.nativeId === `user:${username}`);
 
+        const clusterPeers = new Map<number, Promise<{ clusterPeer: RpcPeer, clusterSecret: string }>>();
+        const ensureClusterPeer = (port: number) => {
+            let clusterPeerPromise = clusterPeers.get(port);
+            if (!clusterPeerPromise) {
+                clusterPeerPromise = (async () => {
+                    const eioPath = 'engine.io/connectRPCObject';
+                    const eioEndpoint = baseUrl ? new URL(eioPath, baseUrl).pathname : '/' + eioPath;
```

Comment:
Should use relative paths. Home assistant will rehost Scrypted's web interface on a sub path.